test/unit/alignment/matrix/detail/affine_cell_proxy_test.cpp

```diff
@@ -0,0 +1,92 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+#include "gtest/gtest.h"
+
+#include <seqan3/alignment/matrix/detail/affine_cell_proxy.hpp>
+
+struct affine_cell_proxy_test : public ::testing::Test
+{
+    using base_t = std::tuple<int, int const &, int &>;
+    using cell_t = seqan3::detail::affine_cell_proxy<base_t>;
+
+    int optimal{4};
+    int horizontal{-1};
+    int vertical{10};
+
+    cell_t affine_cell{optimal, horizontal, vertical};
+    cell_t const const_affine_cell{affine_cell};
+};
+
```

Comment:
I think the concept check `tuple_like` is missing?

Reply:
well I am not explicitly needing it, but I can add the check anyhow